dynamic-distro/scatest/code-from-elsewhere/bad-package.bb

```diff
@@ -0,0 +1,118 @@
+SUMMARY = "A really bad package"
```

Comment:
[Package:bad-po Tool:msgcheck] This should is an example warning ::: occured in file README.md